packages/pysam/patches/0001-use-emmake.patch

```diff
@@ -32,11 +32,11 @@ index 22f3cb3..39da16d 100644
  
  
  def run_make_print_config():
--    stdout = subprocess.check_output(["make", "-s", "print-config"])
-+    stdout = subprocess.check_output(["emmake", "make", "-s", "print-config"])
-     if IS_PYTHON3:
-         stdout = stdout.decode("ascii")
+-    stdout = subprocess.check_output(["make", "-s", "print-config"], encoding="ascii")
++    stdout = subprocess.check_output(["emmake", "make", "-s", "print-config"], encoding="ascii")
```

Comment:
Should this be covered by pywasmcross?

Reply:
I am not sure why pywasmcross was not covering this case. Maybe we can check that later.